### DISCUSSION TITLE
OneBodyAngularMotor has force without leverage?

### DISCUSSION THREAD

**Discussion by harrisse** [bepu/bepuphysics2#181] [Q&A] [answered]
Hello, OneBodyAngularMotor accepts target velocity, max force, and damping. To apply angular velocity I thought it would also have to take an offset in order to convert the motor's force to torque by calculating the length of the lever arm. I see there are some calculations being performed to convert force to impulse -- does this assume the max force in the motor settings is premultiplied by the length of the lever arm (making it max torque) or am I forgetting something from physics class?

**Comment by RossNordby** [comment, marked as answer]
For angular-only constraints, the 'maximum force' in MotorSettings is interpreted as maximum torque. There's no lever arm involved, just a direct effect on angular velocity.

It's an unfortunate naming side effect of reusing MotorSettings across all types of motors.

**Reply by harrisse** [reply]
Makes sense, thanks! I only noticed because I'm writing an F# wrapper and attaching typed units of measure to everything :)
